mkdocs.yml

```diff
@@ -36,6 +36,7 @@ markdown_extensions:
   - markdown.extensions.codehilite:
       guess_lang: false
   - mkautodoc
+  - admonition
```

Comment:
This is to get the nice warning message to render:
<img width="722" alt="Screenshot 2021-02-03 at 22 00 21" src="https://user-images.githubusercontent.com/1748643/106817645-11be0180-666f-11eb-9f46-477dfc5c79b5.png">